src/phases/game-over-phase.ts

```diff
@@ -26,36 +26,36 @@ import i18next from "i18next";
 import { pokerogueApi } from "#app/plugins/api/pokerogue-api";
 
 export class GameOverPhase extends BattlePhase {
-  private victory: boolean;
+  private isVictory: boolean;
   private firstRibbons: PokemonSpecies[] = [];
 
-  constructor(scene: BattleScene, victory?: boolean) {
+  constructor(scene: BattleScene, victory: boolean = false) {
```

Comment:
nit: (might as well change this too
```suggestion
  constructor(scene: BattleScene, isVictory: boolean = false) {
```